src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -83,6 +83,10 @@ private static function getPhpUnitCliConfigArgument()
                 $argPath = substr($testArg, strlen('--configuration='));
                 $dir = realpath($argPath);
                 break;
+            } elseif (strpos($testArg, '-c') === 0) {
```

Comment:
Shouldn't this case be handled by the first condition - `preg_match('/^-[^ \-]*c$/', $testArg)` ?